.github/workflows/cygwin.yml

```diff
@@ -84,16 +86,26 @@ jobs:
             vala
             zlib-devel
 
+      - name: Create cygwin shell wrapper and add to PATH
```

Comment:
If I'm reading this correctly, this is just for the purposes of putting the path and options which we use to invoke bash in several steps in one place.

So a comment to that effect around here would be nice?